tests/multigrid_evolve_test.cpp

```diff
@@ -60,6 +60,31 @@ display_test_info()
   printfQuda("Outer solver paramers\n");
   printfQuda(" - pipeline = %d\n", pipeline);
 
+  if(low_mode_check || use_low_modes || deflate_coarsest) {
```

Comment:
Why are change to multigrid_evolve_test in this pull?

Reply:
Something just went out of whack at some point, I can't explain it. In lieu of thinking about it I grabbed the latest version of `test/multigrid_evolve_test` and added the `is_staggered = QUDA_BOOLEAN_NO` check.